lib/store.ts

```diff
@@ -61,4 +67,12 @@ export const recommendStandardDeduction = (state: RootState) => {
     const benefits = selectBenefits(state)
 
     return (benefits.standardDeduction || benefits.snap || benefits.deductionAmount !== undefined) ? true : false
-}
\ No newline at end of file
+}
+
+/**
+ * Generates uuids for object ids
+ * 
+ * @returns string
+ */
+
+export const createUuid = () => uuidv4()
```

Comment:
i'm not 100% sure this is the right place for this method, but here it will live until i find a better spot